### DISCUSSION TITLE
Where can i change after installation surricata number of process

### DISCUSSION THREAD

**Discussion by xSunShine** [Security-Onion-Solutions/securityonion#9801] [Unsupported Versions]
Hello,

Where can i change number of proccesses for surricata manually after installation distrbued? on which node should i do changes and what file should i change?

Thank you

**Comment by cm-ops** [comment]
You would make those changes in the sensor node minion pillar file - `/opt/so/saltstack/local/pillar/minion/$SENSORNAME_$ROLE.sls` Also, see https://docs.securityonion.net/en/latest/suricata.html?#performance and https://docs.securityonion.net/en/latest/zeek.html#performance
